### ISSUE TITLE
ubuntu electrum wallet 2.4.1 not connected

### ISSUE THREAD

**Issue by croucha** [vergecurrency/VERGE#431]
I'm trying to connect with 2.4.1 and it seems I'm not connected. 

I'm using auto connect and socks5 with SSL 127.0.0.1 and port 9050.

**Comment by vorlonempire** [comment]
e1.verge-electrum.com port 50003
Use SSL check

**Comment by croucha** [comment]
Thanks @vorlonempire,

I assume there are no further steps to connect besides having the correct network settings?

I tried e1.verge-electrum.com:5003 without auto connect and proxy socks5 127.0.0.1:9050 and still no luck.  Strange thing is, after I change domain port to 5003 and close and open again, I notice the port resets to 5001...

**Comment by justinvforvendetta** [comment]
if you are using the electrum tor just run tor and run the wallet and it should connect immediately. I couldn't reproduce this issue

**Comment by croucha** [comment]
Is tor not bundled in with the wallet?  I need to install and run tor first?

**Comment by justinvforvendetta** [comment]
yeah there's instructions on its repository, but basically it's run tor and run electrum, done.

**Comment by croucha** [comment]
Right, I followed them here:

https://github.com/vergecurrency/electrum-xvg-tor

I was thinking there's more involved. I did follow the tor steps in the guide (restart and torrc file changes).

However I assumed it's installed with unbuntu 17.10?

**Comment by justinvforvendetta** [comment]
well its on whatever distro you want but the windows and osx binaries are in releases

**Comment by croucha** [comment]
I'm trying to build source for Ubuntu over here: https://github.com/vergecurrency/VERGE and waiting for the blockchain to download. I don't think it's setup for tor yet?  

Originally, before Ubuntu, I tried this on OSX. I downloaded the OSX tor version "Electrum-XVG-TOR.app" but whenever I open it, I get "Electrum-XVG-TOR quit unexpectedly."  I have OSX El Capitan 10.11.6...

**Comment by justinvforvendetta** [comment]
electrum and this are completely different wallets

**Comment by croucha** [comment]
That's what I thought.  Here are my Network settings on Ubuntu trying to use electrum 2.4.1:

![image](https://user-images.githubusercontent.com/2465008/34803373-99c305a8-f638-11e7-96a2-3550f8a68575.png)

One thing that appears buggy for sure, is once I set those settings and click the "OK" button and I go back to those settings and view them, the port has changed from 50003 to 50002...

![image](https://user-images.githubusercontent.com/2465008/34803438-ed5b0fd0-f638-11e7-973d-b8e88ec6f314.png)

In addition to this, during my install process of sudo pip install pyasn1 pyasn1-modules pbkdf2 tlslite qrcode, I saw the following:

![image](https://user-images.githubusercontent.com/2465008/34803483-303616ce-f639-11e7-84ff-30a4bf017bab.png)

**Comment by croucha** [comment]
So I haven't heard back and while it would be nice to run on Linux I discovered the Wraith Full release for OS X, which is really hard to find (not linked on your site and I found it through reddit).  

https://github.com/vergecurrency/VERGE/releases

Anyway, I had to upgrade my OS X to be 10.12 or greater and it worked! Worked in the sense that it opens and is synching the block.  I will have to try a test run of sending a few coins to it to see how it goes.
